skills_taxonomy_v2/pipeline/skills_extraction/skills_naming.py

```diff
@@ -63,21 +70,40 @@ def parse_arguments(parser):
     s3 = boto3.resource("s3")
 
     # Load data
-    sentence_skills = load_s3_data(s3, BUCKET_NAME, params["sentence_skills_path"])
-    sentence_skills = pd.DataFrame(sentence_skills)
-    sentence_skills = sentence_skills[sentence_skills["Cluster number"] != -1]
-    sentence_embs = load_s3_data(s3, BUCKET_NAME, params["embedding_sample_path"])
-
-    # Find n-grams and get skill information
-    clean_ngrams, cluster_descriptions = get_clean_ngrams(
-        sentence_skills, params["ngram"], params["min_count"], params["threshold"]
+    skill_sentences = load_s3_data(s3, BUCKET_NAME, params["new_skill_sentences_path"])
+    skills_embeds = get_new_skills_embeds(params["new_skills_embeds_path"], BUCKET_NAME)
+    sent_cluster_embeds = load_s3_data(
+        s3, BUCKET_NAME, params["mean_skills_embeds_path"]
+    )
+    skills = load_s3_data(s3, BUCKET_NAME, params["new_skills_path"])
+
+    # wrangle data in the format needed
+    skills_embeds_df = pd.DataFrame(skills_embeds)[
+        ["original sentence", "sentence id", "embedding"]
+    ]
+    skill_sentences_df = pd.DataFrame(skill_sentences)[
+        ["sentence id", "Cluster number predicted"]
+    ]
+    merged_sents_embeds = pd.merge(
+        skills_embeds_df, skill_sentences_df, on="sentence id"
     )
+    merged_sents_embeds = merged_sents_embeds[
+        merged_sents_embeds["Cluster number predicted"] != -2
+    ]
+
+    skills_df = pd.DataFrame(skills).T
+    skills_df["Mean embedding"] = sent_cluster_embeds.values()
```

Comment:
I guess this is reliant on the order of `skill_df` being the same skill number order as `sent_cluster_embeds`. Not sure if this is always going to be the case, so I'd just do something like:
```
skills_df["Mean embedding"] = skills_df["Skill number"].apply(lambda x: sent_cluster_embeds[x])
```
(You may need to create `skills_df["Skill number"]` with the index?)